reemgawad/aircams
# Front end for my listings page

What changed:
-Added: Add a camera button
-Got rid of the rental details on card (will be on another page)
-Made cards pretty
-Linked rental details in "rentals"
-Made the edit link blue (changed hover color as well)

How to test:
-GOTO browser/my_listings(path: my_cameras)--> see if it is beautiful enough
-Click on "rental(s)"
-Click on camera brand
